File: beecrowd/beginner/python/1036.py
from math import sqrt

a, b, c = map(float, input().split())

delta = b**2 - 4 * a * c

if a == 0 or delta < 0:
    print("Impossivel calcular")
else:
    root1 = (-b + sqrt(delta)) / (2 * a)
    root2 = (-b - sqrt(delta)) / (2 * a)
    print(f"R1 = {root1:.5f}")
    print(f"R2 = {root2:.5f}")

File: beecrowd/beginner/c/1036.c
#include <stdio.h>
#include <math.h>

int main(){
    double a, b, c, delta, root1, root2;

    scanf("%lf %lf %lf", &a, &b, &c);

    delta = b * b - 4 * a * c;

    if(delta < 0 || a == 0){
        printf("Impossivel calcular\n");
    }else{
        root1 = (-b + sqrt(delta)) / (2 * a);
        root2 = (-b - sqrt(delta)) / (2 * a);
         
        printf("R1 = %.5lf\n", root1);
        printf("R2 = %.5lf\n", root2);
    }

    return 0;
}
